test/e2e/bandwidth_test.go

```diff
@@ -25,6 +25,7 @@ import (
 )
 
 const iperfPort = 5201
+const iperfSvcPort = 9999
```

Comment:
I see that `iperfPort` is used by several test files. Maybe it makes more sense to move both declarations to framework.go.